web/experimental/new_embed.css

```diff
@@ -82,6 +97,10 @@ body {
   border-left: 1px solid #e1e4e8;
 }
 
+#user-code-editor.web-output-showing {
+  width: calc(100% - 260px);
```

Comment:
Way out of scope here, but I've been wondering if we'll eventually need to let the gist or a query string parameter determine the width of the web output view. Some codelab authors might need to guarantee a certain number of pixels.

Reply:
We could do as a query param. There's also some value is just having fewer options, and encouraging authors to be creative within those constraints.